app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const app = express()
+const get = express()
```

Comment:
Get is a method so we don't need to explicitly require it

Reply:
Thank you Mr. Swanson